sdks/python/apache_beam/testing/util.py

```diff
@@ -88,6 +107,7 @@ def _equal(actual):
     if sorted_expected != sorted_actual:
       raise BeamAssertException(
           'Failed assert: %r == %r' % (sorted_expected, sorted_actual))
+
```

Comment:
<!--new_thread; commit:1d8257d6902c54d33062c23cce99e06fbf42c41d; resolved:0-->
Could you remove this extra line?

Reply:
Thanks. The linter didn't catch this either.